src/coreComponents/physicsSolvers/solidMechanics/SolidMechanicsStatistics.hpp

```diff
@@ -0,0 +1,98 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 TotalEnergies
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file SolidMechanicsStatistics.hpp
+ */
+
+#ifndef SRC_CORECOMPONENTS_PHYSICSSOLVERS_SOLIDMECHANICS_SOLIDMECHANICSSTATISTICS_HPP_
+#define SRC_CORECOMPONENTS_PHYSICSSOLVERS_SOLIDMECHANICS_SOLIDMECHANICSSTATISTICS_HPP_
+
+#include "physicsSolvers/FieldStatisticsBase.hpp"
+
+namespace geosx
+{
+
+class SolidMechanicsLagrangianFEM;
+
+/**
+ * @class SolidMechanicsStatistics
+ *
+ * Task class allowing for the computation of aggregate statistics in solid mechanics simulations
+ */
+class SolidMechanicsStatistics : public FieldStatisticsBase< SolidMechanicsLagrangianFEM >
+{
+public:
+
+  using Base = FieldStatisticsBase< SolidMechanicsLagrangianFEM >;
```

Comment:
Do you need this? And do you need it `public`?

Reply:
It is needed in the constructor of the class for instance. I made it `private`.

Reply:
I was able to spot its usage, but since you use it only once in the ctor, I was thinking it was a bit overkill. Anyway, `private` is better.